FNPlugin/Collectors/SolarWindCollector.cs

```diff
@@ -130,9 +132,22 @@ public override void OnFixedUpdate()
                 if (!bIsEnabled)
                 {
                     strCollectingStatus = "Disabled";
-                    strStarDist = (CalculateDistanceToSun() / 1000).ToString("F2") + " km"; // trying to solve the bullshit
+                    strStarDist = UpdateDistanceInGUI(); // passes the distance to the GUI
+                    return;
+                }
+
```

Comment:
Notice that earth magnetosphere also affect solar particle density in a big way

http://www.astrobio.net/images/galleryimages_images/Gallery_Image_7460.jpg

Although it is impossible to determin the extend of the magnetophere, we can make some gross generalisations like  that within the magnetosphere there is no solar wind and it extends 10 times the hight of the atmosphere ...